### PR TITLE
refactor: badge improvements

### PR DESCRIPTION
**Pull request by qouv**

### CODE REVIEW

File: huddleupui/src/layouts/CommunityLayout.jsx [hunk 1]
```diff
@@ -148,295 +149,225 @@ export default function CommunityLayout({ children, allowedUserTypes, canNotMemb
...
+																	style={{ filter: badge.userHasBadge ? 'none' : 'blur(4px)' }}
+																/>
+																<div>{badge.name}</div>
+															</Tooltip>
+														</Col>
+													))}
+												</Row>
+												{communityInfo.badges.length > 10 && (
+													<Row justify="center">
+														<span style={{ color: 'blue', cursor: 'pointer' }} onClick={() => setShowMoreBadges(true)}>Show More Badges</span>
```
Comment (ovuncertunc) [inline huddleupui/src/layouts/CommunityLayout.jsx:291]: Did you define setShowMoreBadges function in somewhere? it gives error 'setShowMoreBadges' is not defined

Comment (qouv) [reply huddleupui/src/layouts/CommunityLayout.jsx:292]: you are right, fixed

File: huddleupui/src/layouts/CommunityLayout.jsx [hunk 2]
```diff
@@ -530,8 +563,107 @@ export default function CommunityLayout({ children, allowedUserTypes, canNotMemb
 									</Row>
 								</Card>
 							)}
-						</Drawer>
```
Comment (ovuncertunc) [inline huddleupui/src/layouts/CommunityLayout.jsx:533]: this is community sidebar drawer closing, you shouldnt delete it

Comment (qouv) [reply huddleupui/src/layouts/CommunityLayout.jsx]: i think it is ok now?

**Pull request merged**